src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images\star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars_star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--1 .stars_star:nth-child(-n + 1),
+.stars--2 .stars_star:nth-child(-n + 2),
+.stars--3 .stars_star:nth-child(-n + 3),
+.stars--4 .stars_star:nth-child(-n + 4),
+.stars--5 .stars_star:nth-child(-n + 5) {
+  background-image: url(images\star-active.svg);
```

Comment:
The file path in CSS should use forward slashes, not backslashes. Replace 'images\star-active.svg' with 'images/star-active.svg'.

Reply:
VS Code made file path with backslashes automatically. Corrected it manually.
<img width="688" alt="Zrzut ekranu 2023-07-27 o 11 32 59" src="https://github.com/mate-academy/layout_stars/assets/58208742/35e9dbd1-106e-4044-bdd0-dc3e979805c4">